packages/palette/src/elements/EntityHeader/EntityHeader.tsx

```diff
@@ -52,35 +54,68 @@ export const EntityHeader: SFC<EntityHeaderProps> = ({
         </Flex>
       )}
 
-      <Flex flexDirection="column" justifyContent="center" width="100%">
-        <Sans size="3" weight="medium" color="black100">
-          {name}
-        </Sans>
+      {small ? (
+        <Flex alignItems="center" width="100%">
+          <Sans size="3">{name}</Sans>
 
-        <Sans size="2" color="black60">
-          {!!meta && <span>{meta}</span>}
+          <Sans size="3">
+            {FollowButton && (
+              <>
+                {
+                  <Sans size="3" mx={0.3} display="inline-block">
+                    •
+                  </Sans>
+                }
+                <Box
+                  display="inline-block"
+                  onClick={event => {
+                    // Capture click event so that interacting with Follow doesn't
+                    // trigger Container's link.
+                    event.stopPropagation()
+                  }}
+                >
+                  {FollowButton}
+                </Box>
+              </>
+            )}
+          </Sans>
+        </Flex>
+      ) : (
+        <Flex flexDirection="column" justifyContent="center" width="100%">
+          <Sans size="3" weight="medium" color="black100">
+            {name}
+          </Sans>
+
+          <Sans size="2" color="black60">
+            {!!meta && <span>{meta}</span>}
 
-          {FollowButton && (
-            <>
-              {meta && (
-                <Sans size="2" color="black60" mx={0.3} display="inline-block">
-                  •
-                </Sans>
-              )}
-              <Box
-                display="inline-block"
-                onClick={event => {
-                  // Capture click event so that interacting with Follow doesn't
-                  // trigger Container's link.
-                  event.stopPropagation()
-                }}
-              >
-                {FollowButton}
-              </Box>
-            </>
-          )}
-        </Sans>
-      </Flex>
+            {FollowButton && (
+              <>
+                {meta && (
+                  <Sans
+                    size="2"
+                    color="black60"
+                    mx={0.3}
+                    display="inline-block"
+                  >
+                    •
+                  </Sans>
+                )}
+                <Box
+                  display="inline-block"
+                  onClick={event => {
+                    // Capture click event so that interacting with Follow doesn't
+                    // trigger Container's link.
+                    event.stopPropagation()
```

Comment:
I can see this was here all ready but this just suggests to me that the structure of this component is odd — like is it appropriate UX to extend the hit area to the right of a borderless follow button? 🤔

Reply:
But the Sans component has a div wrapper, so there are bound to the follow button even when its text. Is the suggestion here to add an invisible border around the FollowButton when its text and not an actual Button component?

Reply:
Ah, I had assumed that this was an anchor/a tag with a nested button. Tapping the button would propagate the parent link click. Rather I suppose would be to link everything separately and the follow button as an actual button would be fine and wouldn't have to have an outer wrapper prevent the event propagation. I'd say just leave this alone, I guess just making a mental note.